### ISSUE TITLE
Add survey task and annotation validations

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/front-end-monorepo#1987]
Using [`types.refinement`](https://mobx-state-tree.js.org/API/#refinement),

survey task validations:
- [x] characteristics
- [ ] choices
- [ ] questions
- [ ] questionsMap

survey annotation validations:
- [ ] answers
- [ ] filters

Iteration on #1935.

**Comment by goplayoutside3** [comment]
@mcbouslog Is this still something to be added to the classifier? Is this the correct related file? https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/survey/models/SurveyTask.js

**Comment by mcbouslog** [comment]
Yes, additional validations could be added, and that is indeed the correct related file (https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/survey/models/SurveyTask.js). In that file the `characteristics` property has been validated, but the other properties noted in this issue's initial comment have not been validated. However, I think there might be many similar situations (unvalidated properties/objects) throughout the code that don't have a related issue.
